packages/react-composites/src/composites/CallComposite/components/MediaGallery.tsx

```diff
@@ -36,16 +36,16 @@ export interface MediaGalleryProps {
   onStartLocalVideo: () => Promise<void>;
   onRenderAvatar?: OnRenderAvatarCallback;
   onFetchAvatarPersonaData?: AvatarPersonaDataCallback;
+  /* @conditional-compile-remove-from(stable) */
+  isMobile?: boolean;
```

Comment:
Why is this conditionally compiled? I'm guessing this is driven by the `CallComposite` `mobileView` flag, which is already known in stable build, and you can simply not-use it in stable build in side `MediaGallery`. Try to make this not be conditionally compiled.

Reply:
okay I will look into that!